### PR TITLE
General HTML for puzzle page(with old sources)

### PR DESCRIPTION
**Pull request by kblueberry**
image

General overview of page

### CODE REVIEW

**Review by SeriyBg — COMMENTED** [review]
Shouldn't `node_modules` be in `.gitignore`? Having 5000+ files changed in PR is wild :)

**Comment by kblueberry** [comment]
> Shouldn't `node_modules` be in `.gitignore`? Having 5000+ files changed in PR is wild :)

Good point, didn't think about it

**Comment by kblueberry** [comment]
Added node_module to gitignore

File: frontend/src/components/PuzzlePage/PuzzlePage.css [hunk 1]
```diff
@@ -1,7 +1,10 @@
+* {
+  font-family: Roboto;
```
Comment (mediumwd) [inline frontend/src/components/PuzzlePage/PuzzlePage.css]: By design font-family: Montserrat;

**Review by sergiykatriuk — APPROVED** [review]

**Comment by SeriyBg** [comment]
> Added node_module to gitignore

@kblueberry can you delete node_modules from the PR? It's still the PR with 5000+ files changed :)

**Comment by kblueberry** [comment]
Not actual already

**Pull request closed without merging**